news/2024/2024-11-28-osu-world-cup-2024-semifinals-recap.md

```diff
@@ -0,0 +1,135 @@
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116290548)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2309726416)**
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/udSXyASuYI4?start=1021&end=1184" frameborder="0" allowfullscreen></iframe>
+</div>
+
+The moment the Qualifier results were released, with ::{ flag=AU }:: Australia seeding #2 and the ::{ flag=US }:: United States seeding #3, all eyes looked toward their near-certain date in the Semifinals. After several weeks of one-sided wins each, their meeting was set, a rematch of last year's losers' bracket Finals in which the United States ran roughshod over Australia 7–1.
+
+Expectations for this match were sky-high, and it lived up to the billing, a back-and-forth affair in which the teams dominated their own picks to a historic level. The ::{ flag=US }:: United States, having bolstered their roster's well-roundedness with the return of ::{ flag=US }:: [Utami](https://osu.ppy.sh/users/7512553) and the addition of rookie ::{ flag=US }:: [Pezz](https://osu.ppy.sh/users/10651106), opted heavily into the gimmicky maps in the pool, opening things up with the overlap-reading No Mod pick [Lovely Icecream Princess Sweetie](https://osu.ppy.sh/beatmapsets/669232#osu/1415526) before delving into the precision flow aim Hard Rock pick [Saika](https://osu.ppy.sh/beatmapsets/1224300#osu/2546419). They would win both with ::{ flag=US }:: [Pezz](https://osu.ppy.sh/users/10651106) and ::{ flag=US }:: [Utami](https://osu.ppy.sh/users/7512553) being their top performers.
+
+Not to be outdone, the mechanics marvels of ::{ flag=AU }:: Australia simply decided to pick their way straight through the Double Time pool, and the Aussies would be rewarded with wins on all four DT maps, including a ridiculous 4.39 million team score on [Hit & Run](https://osu.ppy.sh/beatmapsets/2281450#osu/4864105). Back and forth went the scoreline as neither team could pick up a break point through the first eight maps of the match, bringing the score to 4–4 heading into the teams' respective final picks.
+
+After the US won their last map, the last pick of the match belonged to Australia. If they won, it would force a tiebreaker; lose, and the match would end right then and there with an American victory. Out of the Double Time maps, they opted into the aim consistency No Mod selection of [Warcry](https://osu.ppy.sh/beatmapsets/2281532#osu/4864391), an incredibly popular pick for the entire weekend. To this point, ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) had been having something of a quiet match by his illustrious standards — while he put up several full combos and top scores, none truly stood out in the way many expected. As one of the pre-eminent aim players in the tournament, what would NM1 bring for him?
```

Comment:
```suggestion
After the US won their last map, the last pick of the match belonged to ::{ flag=AU }:: Australia. If they won, it would force a tiebreaker; lose, and the match would end right then and there with an ::{ flag=US }:: American victory. Out of the Double Time maps, they opted into the aim consistency No Mod selection of [Warcry](https://osu.ppy.sh/beatmapsets/2281532#osu/4864391), an incredibly popular pick for the entire weekend. To this point, ::{ flag=AU }:: [mrekk](https://osu.ppy.sh/users/7562902) had been having something of a quiet match by his illustrious standards — while he put up several full combos and top scores, none truly stood out in the way many expected. As one of the pre-eminent aim players in the tournament, what would NM1 bring for him?
```